homeassistant/components/airly/sensor.py

```diff
@@ -99,19 +101,35 @@ def __init__(self, airly, name, kind, unique_id):
         self._unit_of_measurement = None
         self._attrs = {ATTR_ATTRIBUTION: ATTRIBUTION}
 
+    async def async_added_to_hass(self):
+        """Call when entity is added to HA."""
+        self._async_unsub_dispatcher_connect = async_dispatcher_connect(
+            self.hass, TOPIC_DATA_UPDATE, self._update_callback
+        )
+        self._update_callback()
+
+    async def async_will_remove_from_hass(self):
+        """Disconnect dispatcher listener when removed."""
+        if self._async_unsub_dispatcher_connect:
+            self._async_unsub_dispatcher_connect()
+        # pylint: disable=protected-access
```

Comment:
Is this work around needed?

Reply:
Without disabled `protected-access` there are pylint warnings W0212 for lines 116, 117 and 118.